Scripts/Python/QueryGroupManagementOmeREDFISH.py

```diff
@@ -0,0 +1,706 @@
+#
+# QueryGroupManagementOmeREDFISH. Python script using Redfish API with OEM extension to manage Open Manage Enterprise Query Groups
+#
+# _author_ = Texas Roemer <Texas_RoemerDell.com>
+# _version_ = 1.0
+#
+# Copyright (c) 2020, Dell, Inc.
+#
+# This software is licensed to you under the GNU General Public License,
+# version 2 (GPLv2). There is NO WARRANTY for this software, express or
+# implied, including the implied warranties of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+# along with this software; if not, see
+# http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+#
+
+
+
+
+import requests, json, sys, re, time, warnings, argparse, os
+
+from datetime import datetime
+
+warnings.filterwarnings("ignore")
+
+parser = argparse.ArgumentParser(description='Python script using Redfish API with OEM extension to create/update/delete Open Manage Enterprise Query Groups')
```

Comment:
Non functionality affecting but would it be possible to have consistent semantics for arguments - use --user or -u instead of having some arguments specified with -- and some with -

Unless I missed something and the ones specified with the -- have a different context

 PARSER.add_argument("--user", "-u", required=False...